plugin.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "excel-sheet-importer",
-    "version": "1.0.1",
+    "version": "1.0.2",
```

Comment:
It looks like we have to update this in the makefile also - currently it is building 1.0.1